src/App.test.js

```diff
@@ -1,8 +1,126 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import Calculate from './logic/calculate';
+import Operate from './logic/operate';
```

Comment:
- [ ] In `operate.js`, we have some senarios we need to test for.
1. First we need to add a test which checks that a number cannot be divided by `o`
![image](https://user-images.githubusercontent.com/81932478/236161833-ef24b823-eee5-4de9-8312-3e47cb73b130.png)

2. Scond we need two tests for the `%` modulus operator, first, we need to test that it returns the correct value, and second that it cannot return the modulo as you cannot divide a number by zero
![image](https://user-images.githubusercontent.com/81932478/236161084-4b9883fc-f9f9-4f19-bc02-95f92fad6257.png)

3. Third we have to test for an `unknown` mathematical operation 
![image](https://user-images.githubusercontent.com/81932478/236160983-a2948edc-8812-4016-9b14-e2ec7e88f67e.png)